aws-py-eks/vpc.py

```diff
@@ -0,0 +1,104 @@
+from pulumi_aws import ec2
+
+REGION = "us-east-2"
```

Comment:
We shouldn't hardcode this value

Reply:
This is now pulling region from the config